### PR TITLE
[ADD] module l10n_ch_invoice_with_payment

### PR DESCRIPTION
**Pull request by damdam-s**
[ADD] module l10n_ch_invoice_with_payment

### CODE REVIEW

**Review by Tonow-c2c — COMMENTED** [review]
To update the readme use [https://github.com/OCA/maintainer-tools/#readme-generator](https://github.com/OCA/maintainer-tools/#readme-generator)

File: l10n_ch_invoice_with_payment/README.rst [hunk 1]
```diff
@@ -0,0 +1,21 @@
+**This file is going to be generated by oca-gen-addon-readme.**
```
Comment (asaunier) [inline l10n_ch_invoice_with_payment/README.rst]: This content is not the expected one :P 
See @Tonow-c2c 's comment about README generating.

File: l10n_ch_invoice_with_payment/readme/DESCRIPTION.rst [hunk 2]
```diff
@@ -0,0 +1,2 @@
+This module allows the user to print an invoice and a payment slip in the same pdf
```
Comment (asaunier) [inline l10n_ch_invoice_with_payment/readme/DESCRIPTION.rst]:
```suggestion
This module allows the user to print an invoice and a payment slip in the same PDF.
```

**Review by asaunier — COMMENTED** [review]
A small issue with the README, else LGTM

**Review by Tonow-c2c — APPROVED** [review]

File: l10n_ch_invoice_with_payment/__manifest__.py [hunk 3]
```diff
@@ -0,0 +1,20 @@
+# Copyright 2019 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+{
+    'name': 'Invoice report with payment',
+    'version': '11.0.1.0.0',
+    'author': 'Camptocamp, Odoo Community Association (OCA)',
+    'license': 'AGPL-3',
+    'category': 'Accounting',
+    'website': 'http://www.camptocamp.com',
+    'images': [],
```
Comment (i-vyshnevska) [inline l10n_ch_invoice_with_payment/__manifest__.py:10]:
```suggestion
```

**Review by i-vyshnevska — APPROVED** [review]

**Pull request merged**